### PR TITLE
Activated appeal scenarios

### PR DESCRIPTION
**Pull request by bshyong**
Add scenarios for `activated` event

resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3221

screen shot 2017-06-28 at 10 58 18 am
screen shot 2017-06-28 at 10 58 04 am
screen shot 2017-06-28 at 10 57 52 am

### CODE REVIEW

**Review by U-DON — APPROVED** [review]
LGTM.

**Comment by U-DON** [comment]
Build seems to be failing on a broken link from merging in the `health-care` link updates from master.

File: src/js/disability-benefits/utils/appeal-helpers.jsx [hunk 1]
```diff
@@ -157,20 +182,31 @@ export function appealStatusDescriptions(lastEvent, previousHistory = []) {
...
+      return {
+        ...eventContent,
+        status: eventContent.status[prevType || 'defaultStatus'],
+      };
+    default:
+      break;
   }
 
-  return eventContent || emptyResponse;
+  return contentMap[lastEvent.type] || emptyResponse;
```
Comment (bshyong) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: Good catch

Comment (bshyong) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: Good catch — something strange with the merge

**Pull request merged**